internal/app/app.go

```diff
@@ -114,6 +115,20 @@ func (p *PolicyAutomationApp) LoadConfig(config *Config) (err error) {
 			}
 			p.collectors = append(p.collectors, storageCollector)
 		}
+		if len(o.PubSub.Topic) > 0 {
+			var client outputs.PubSubClient
+			if p.config.CredentialsFile != "" {
+				client, err = pbc.NewPubSUbClientWithCredentialsFile(p.ctx, o.PubSub.Project, p.config.CredentialsFile)
+			} else {
+				client, err = pbc.NewPubSUbClient(p.ctx, o.PubSub.Project)
+			}
+			if err != nil {
+				return err
+			}
+			p.collectors = append(p.collectors, outputs.NewPubSubResultCollector(client, o.PubSub.Project, o.PubSub.Topic))
+		} else {
+			p.collectors = append(p.collectors, outputs.NewConsoleResultCollector(p.out))
```

Comment:
this collector is appended at the beginning separately from the Outputs in configuration. Does not need to be added here.

Reply:
Removed it.